src/Lumi/Components2/Banner.purs

```diff
@@ -0,0 +1,231 @@
+                                )
+                            $ _ { content =
+                                    [ case props.icon of
+                                        Just icon ->
+                                          box
+                                          $ S._alignSelf S.Start
+                                          $ S.onDesktop (S._alignSelf S.Center)
+                                          $ style_
+                                              ( fold
+                                                  [ css
+                                                      { marginRight: int 16
+                                                      }
+                                                  , desktopQuery $ css
+                                                      { alignSelf: str "center"
+                                                      }
+                                                  ]
+                                              )
+                                          $ _ { content = [ icon ]
+                                              }
+                                        _ -> mempty
+                                    ]
+                                    <> props.content
+                                }
+                          ]
+                      }
+                ,  Monoid.guard props.dismissable
+                    $ E.element R.div'
+                      { children:
+                          [ Icon.icon
+                              { style: R.css { fontSize: "12px" }
+                              , type_: Icon.Remove
+                              }
+                          ]
+                      , className: ""
+                      , css: theme # toCSS dismissButtonStyle
+                      , onClick: capture_ $ setVisible \_ -> false
+                      }
+                ]
+            }
+  where
+    defaults :: Record BannerProps
+    defaults =
+      { component: Banner
+      , title: Nothing
+      , content: []
+      , dismissable: false
+      , icon: Nothing
+      }
+
+    dismissButtonStyle :: forall props. PropsModifier props
+    dismissButtonStyle =
+      S._interactive
+      <<< style \(LumiTheme { colors }) ->
+            css
+              { position: str "absolute"
+              , top: int 16
+              , right: int 24
+              , "&:hover": nested $ css
+                  { color: color colors.black1
+                  }
+              }
+
+actionBanner :: Array JSX -> PropsModifier BannerProps
+actionBanner actions f =
+  f >>> \props -> props
+    { content =
+        [ box
+          $ S._column
+          $ style_
+              ( css
+                { flex: str "1" }
+              )
+          $ S.onDesktop (S._row)
+          $ style_
+              ( fold
+                  [ css
+                      { alignItems: str "flex-end"
+                      }
```

Comment:
This can be written as `S._align S.End` too. I think we should prefer writing styling with style modifiers instead of with CSS, when possible. Same for the `alignItems` below.